### PR TITLE
BDOG-914 starts assetsfrontend with shell, bump to 2.0.4

### PR DESCRIPTION
**Pull request by christopherjturner**

### CODE REVIEW

File: servicemanager/service/smpythonservice.py [hunk 1]
```diff
@@ -100,8 +100,8 @@ def _start_from_binary(self):
         makedirs_if_not_exists("logs")
         with open("logs/stdout.txt", "wb") as out, open("logs/stderr.txt", "wb") as err:
             return subprocess.Popen(
-                cmd_with_params[0].split(),
-                shell=False,
+                cmd_with_params[0],
```
Comment (anshulbajpai) [inline servicemanager/service/smpythonservice.py:103]: It would be nice to leave a comment, why it is done this way.

Comment (christopherjturner) [reply servicemanager/service/smpythonservice.py:103]: the shell flag was to enable the assets_frontend process to run with the same version of python sm was run with which makes it easier to reason about how to start and monitor the process

**Comment by christopherjturner** [comment]
decided against this change. too much of a edge-case, we'd rather they install python3 as default rather than support running sm in py3 with py2 as being default

**Comment by nigelhp** [comment]
Have re-opened this as it would seem to successfully address an issue that is being encountered with 2.0.3 where we are unable to successfully start profiles that contain ASSETS_FRONTEND (see BDOG-1027).

**Comment by nigelhp** [comment]
Following discussion we have decided to go with a different approach, and explicitly check what version 'python' will resolve to.

**Pull request closed without merging**